src/vnx.rs

```diff
@@ -190,6 +190,83 @@ pub struct DeviceCounter {
     out: u64,
 }
 
+#[test]
+fn test_cifs_servers_parser() {
+    use std::fs::File;
+    use std::io::Read;
+
+    let data = {
```

Comment:
If you want you can compress this down to an `include_str!("tests/vnx/cifs_server_query.xml")`